### DISCUSSION TITLE
Figma For Uno--Is It Free Too?

### DISCUSSION THREAD

**Discussion by JimLittle8** [unoplatform/uno#10953] [Q&A] [answered]
I am very confused about Figma's role in Uno development:

1. Is Figma for Uno free, without limitations?
2. Does using Figma for Uno mean that I don't need to design in XAML or use a XAML WYSIWYG editor, like Blend? 
3. If Figma for Uno outputs XAML, can that modified XAML be reimported to Figma and modified (i.e., is round-tripping supported)?

Thank you!

**Comment by sasakrsmanovic** [comment, marked as answer]
Hello @JimLittle8 

1. Our Figma tooling is an add on to the core free open-source Uno Platform offering. It is not needed for cross-platform development with Uno Platform but we believe it can bring 10x productivity for design-to-develop handoff.  It is currently in preview (preview 4) and free to use in any project or any size company. We are not sure yet if it will be open-source, free, or paid product; that really depends on where our community and clients take us to. I invite you to watch the announcement at our Uno Conf to get the scoop. https://youtu.be/QBQvE4M04fQ?t=1927 . I zoomed in to that particular segment of the keynote, but if you have 45 min, I recommend you watch it all. 

2. Yes, you do not need to design in XAML; you can just export it through our Plugin. We have a number of tutorials which show this - Here is a 4-part tutorial on recreating Goodreads app; many others are on our blogs and docs https://platform.uno/blog/from-figma-to-visual-studio-adding-back-end-logic-to-goodreads-app/ and we are coming up with new ones shortly. 

3. In current preview round-tripping is not supported. If you watch the keynote I provided above you will see why we decided to draw the line between design and development at this point. We believe the current approach is pragmatic and we will move that boundary more towards design or development depending on where our community and clients want us to go. Round tripping has been brought up to us by other community members, so we will see how to best execute on this front. I'd love to work with you on this, so if you can please reach out at sasha at platform.uno

**Comment by JimLittle8** [comment]
This is a most excellent--and most appreciated--answer. I will read the links you cited and watch the videos. I will undoubtedly have some follow-up questions, but I am very interested in Uno and in learning more about how to optimize the developer experience by leveraging such tools as Figma, Xaml editors (such as Blend), and Visual Studio. Thank you!

**Reply by sasakrsmanovic** [reply]
Thanks @JimLittle8 , yes please, fire away questions. If possible, please use one post per question. It makes it easier for the questions to be indexed and later found by others who have same/similar questions.
